### DISCUSSION TITLE
Modify your extension so it’s compatible with the version Business Central SaaS Field conflict

### DISCUSSION THREAD

**Discussion by abrarShaikh273** [microsoft/ALAppExtensions#26109] [Q&A]
Hi All,

We have two fields in different extensions in Business Central SaaS for same table one is "Account Manager" and second is "Account_Manager" and we receive email for compatibility issue:
**Extension validation result: 
AL%20Code/Table%20Ext/50100-Customer.TableExt.al (6,22) - Error AL0757: The name of Field 'Account Manager' conflicts with Field 'Account_Manager' defined in TableExtension 'CustomerExt' by the extension 'Customizations by PublisherA (1.0.0.19)'. Choose another name for one of them. Otherwise, this might cause runtime issues**

What is criteria MS follow for field name difference. because in this scenario there is a difference of underscore symbol.

What is the best way to handle this situation because changing field name could lead to breaking changes can we use Force Sync when uploading extension? and we need to data to remains
<img width="288" alt="image" src="https://github.com/microsoft/ALAppExtensions/assets/57489444/fc7c19bb-8026-49c8-8a18-f820bcdd698b">

Can we handle this situation by obsoleting field?

Please suggest. I appreciate any feedback.

**Comment by JesperSchulz** [comment]
Sorry, we'd like to help, but I fear this isn't the right place to ask this question. Try posting your question to the following resources instead: 

[Business Central Community](https://community.dynamics.com/forums/thread/?groupid=e78817ab-a926-4d31-96cc-aef040a4eb04)
[mibuso forum](https://forum.mibuso.com/)
[Viva Engage](https://www.yammer.com/dynamicsnavdev)
